### PR TITLE
Localize C# Interactive window title

### PR DESCRIPTION
**Pull request by tmat**
Fixes [AB#1631087](https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_workitems/edit/1631087)

### CODE REVIEW

File: src/VisualStudio/CSharp/Impl/Interactive/CSharpVsInteractiveWindowProvider.cs [hunk 1]
```diff
@@ -58,8 +58,7 @@ internal sealed class CSharpVsInteractiveWindowProvider : VsInteractiveWindowPro
 
         protected override Guid Id => CSharpVsInteractiveWindowPackage.Id;
 
-        // Note: intentionally left unlocalized (we treat these words as if they were unregistered trademarks)
```
Comment (Cosifne) [inline src/VisualStudio/CSharp/Impl/Interactive/CSharpVsInteractiveWindowProvider.cs:61]: Change is good to me, but is there any background information about this comment?

Comment (tmat) [reply src/VisualStudio/CSharp/Impl/Interactive/CSharpVsInteractiveWindowProvider.cs:61]: I don't remember. Couldn't find anything. Even if we wanted to keep "Interactive" in English it should be in the loc file with explicit note.

Comment (Cosifne) [reply src/VisualStudio/CSharp/Impl/Interactive/CSharpVsInteractiveWindowProvider.cs:61]: I found this https://github.com/dotnet/roslyn/pull/5808
@KevinH-MS @genlu  Do you remember why this is not localized?

**Review by sharwell — CHANGES_REQUESTED** [review]
Pending resolution of https://github.com/dotnet/roslyn/pull/69066/files#r1265770714

**Comment by tmat** [comment]
@sharwell As the AzDO issues shows F# interactive has the window title localized. I think we should do the same.

**Comment by Cosifne** [comment]
> @sharwell As the AzDO issues shows F# interactive has the window title localized. I think we should do the same.

I prefer to wait for @genlu to come back from vacation and check this.

File: src/VisualStudio/CSharp/Impl/Interactive/CSharpVsInteractiveWindowProvider.cs (hunk 1 above)
Comment (genlu) [reply src/VisualStudio/CSharp/Impl/Interactive/CSharpVsInteractiveWindowProvider.cs:61]: I don't remember either. From a pure functionality standpoint, I don't see why we don't want to localize it.

**Review by genlu — APPROVED** [review]

**Pull request merged**